docs/usage.rst

```diff
@@ -8,6 +8,14 @@ First, make sure you're using Python 3,
 you've :doc:`installed the bigchaindb_driver Python package <quickstart>`,
 and you've :doc:`connected to a BigchainDB node or cluster <connect>`.
 
+For the sake of these examples, we'll assume:
+
+.. ipython::
+
+    In [0]: from bigchaindb_driver import BigchainDB
+
+    In [0]: bdb = BigchainDB('http://bdb-server:9984/api/v1')
```

Comment:
> Why do we need to assume that it's 'http://bdb-server:9984/api/v1'?

No particular reason, aside from that's what it was before, and that's what another example uses later in the document. We could also use the default node, via `BigchainDB()`, so it's less confusing?